src/regex/Regex.java

```diff
@@ -1,41 +1,88 @@
 package regex;
 
 import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 class Regex {
     public static void main(String[] args) {
 
         // 1. What does the following pattern match? (\d){36} explain in a println() statement.
 
-        System.out.println("The following pattern ('\\d'){36} matches : ...");
+        System.out.println("The following pattern ('\\d'){36} matches : 36 of any digit");
 
 
 
         String[] TEKmentors = {"Amir Yunas","Mark Bennet", "Rosa Garcia", "Desaree Byers", "Abram Jablonski", "Dylan Fellows", "Emilios Papas", "Jonathan Diamond"};
 
         // 2. Create a new array of the first names of the TEKmentors.  Use Regex to only grab the first name of every TEKmentor.  Push the values to a new array
-        String[] tekMentors;
+        String patternString = "^[^\\s]+";
+        Pattern pattern = Pattern.compile(patternString);
+        Matcher m;
+        String[] tekMentors = new String [TEKmentors.length];
+        for (int i = 0; i < TEKmentors.length; i++){
+            m = pattern.matcher(TEKmentors[i]);
+
+            if(m.find()){
+                tekMentors[i] = m.group(0);
+            } else{
+                System.out.println("No matches found.");
+            }
+        }
+
+        System.out.println(Arrays.toString(tekMentors));
+    
 
 
         // 3. Find all the occurences of any form of 'book' in the following paragraph. use regex to match the occurences and store the count of books in an int.
         
         int bookCount = 0;
+        final String regex = "book";
```

Comment:
You never used bookCount, just incremented it. Display it cuz it's kinda the answer. Other than that great job on this assignment